### PR TITLE
NavigationView Overflow Items Minor UI Fixes

### PR DESCRIPTION
**Pull request by hassaanmasood**
## Motivation and Context
The change is required to align the UI of Overflow Items with other elements in WinUI such as Menus and CommandBar Overflow.

## How Has This Been Tested?
Tested locally with small sample app. Screenshots attached.

## Screenshots (if appropriate):
NavigationViewMinorIssues
NavigationViewMinorIssuesFixed

### CODE REVIEW

File: dev/NavigationView/NavigationView.xaml [hunk 1]
```diff
@@ -363,7 +363,7 @@
                                                     <Style TargetType="FlyoutPresenter">
                                                         <Setter Property="Padding" Value="{ThemeResource TopNavigationViewOverflowMenuPadding}" />
                                                         <!-- Set negative top margin to make the flyout align exactly with the button -->
```
Comment (StephenLPeters) [inline dev/NavigationView/NavigationView.xaml:365]: Can you remove the comment that no longer applies.

Comment (hassaanmasood) [reply dev/NavigationView/NavigationView.xaml]: Done.

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml [hunk 2]
```diff
@@ -1217,7 +1217,10 @@
                 <ControlTemplate TargetType="primitives:NavigationViewItemPresenter">
                     <Grid
                         x:Name="LayoutRoot"
-                        Height="36" 
+                        Height="36"
+				  Margin="{StaticResource NavigationViewItemButtonMargin}"
```
Comment (StephenLPeters) [inline dev/NavigationView/NavigationView_rs1_themeresources.xaml:1221]: Can you fix the white space here

**Comment by StephenLPeters** [comment]
@chigy These updates look like improvements to me. Can you take a look and sign off if the are appropriate?

File: dev/NavigationView/NavigationView.xaml [hunk 3]
```diff
@@ -362,8 +362,8 @@
                                                 <Flyout.FlyoutPresenterStyle>
                                                     <Style TargetType="FlyoutPresenter">
                                                         <Setter Property="Padding" Value="{ThemeResource TopNavigationViewOverflowMenuPadding}" />
-                                                        <!-- Set negative top margin to make the flyout align exactly with the button -->
-                                                        <Setter Property="Margin" Value="0,-4,0,0" />
+                                                        
+                                                        <Setter Property="Margin" Value="0,0,0,0" />
```
[the remark this answers is not in the document]
Comment (hassaanmasood) [reply dev/NavigationView/NavigationView.xaml]: Removed

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml (hunk 2 above)
Comment (hassaanmasood) [reply dev/NavigationView/NavigationView_rs1_themeresources.xaml]: Fixed white spaces

**Comment by hassaanmasood** [comment]
@chigy Made the changes as asked by @StephenLPeters. Have a look please.

**Review by StephenLPeters — APPROVED** [review]

**Comment by hassaanmasood** [comment]
@StephenLPeters, can you ask someone to check why the azure pipeline got stuck and never completed all checks? The changes are minor, should not break anything.

**Comment by chingucoding** [comment]
@hassaanmasood Sometimes, the CI just times out, not sure why this happens though. Only thing one could do is rerun it.

**Review by raghavthind2005 — APPROVED** [review]

**Pull request merged**